delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -227,12 +227,8 @@ def _prepare_customer(self, picking):
             'ZIP': partner.zip,
             'City': partner.city,
             'Country': partner.country_id.code,
+            'DomicilePostOffice': company.postlogistics_office,
```

Comment:
What if `company.postlogistics_office` is False ?

Reply:
Yep maybe
```suggestion
            'DomicilePostOffice': company.postlogistics_office or "",
```
Would be safer. Or keep the if case...

Reply:
@mmequignon  @TDu  thanks a lot for catching this, fixed code.